sdk/openai/azure-ai-openai/README.md

```diff
@@ -286,6 +290,44 @@ for (ImageLocation imageLocation : images.getData()) {
 
 For a complete sample example, see sample [Image Generation][sample_image_generation].
 
+### Audio Transcription
+The OpenAI service starts supporting `audio transcription` since model `Whisper`. 
+The following code snippet shows how to use the service to transcribe audio.
+
+```java readme-sample-audioTranscription
+String fileName = "{your-file-name}";
+Path filePath = Paths.get("{your-file-path}" + fileName);
+
+byte[] file = BinaryData.fromFile(filePath).toBytes();
+AudioTranscriptionOptions transcriptionOptions = new AudioTranscriptionOptions(file)
+    .setResponseFormat(AudioTranscriptionFormat.JSON);
+
+AudioTranscription transcription = client.getAudioTranscription("{deploymentOrModelId}", transcriptionOptions, fileName);
+
+System.out.println("Transcription: " + transcription.getText());
+```
+For a complete sample example, see sample [Audio Transcription][sample_audio_transcription].
+Please refer to the service documentation for a conceptual discussion of [Whisper][microsoft_docs_whisper_model].
+
+### Audio Translation
+The OpenAI service starts supporting `audio translation` since model `Whisper`.
```

Comment:
I think I would aso change here to my previous suggestion. "with the introduction of `Whisper` models"